frontends/krita/krita_comfy/client.py

```diff
@@ -257,10 +221,11 @@ def on_image_received(img):
                 assert img is not None, "Backend Error, check terminal"
                 qimage = QImage()
                 qimage.loadFromData(img)
+                # TODO pass QImage directly
```

Comment:
The reason why we encoded to base64 here is because script's `img_inserted` expects encoded image data. I think you're already handling that in `PromptResponse` class, so it should be good if we just pass the raw data directly.